spec/controllers/stories_controller_spec.rb

```diff
@@ -83,10 +83,43 @@
     end
 
     describe '#index' do
-      specify do
-        get :index, xhr: true, params: { project_id: project.id }
-        expect(response).to be_successful
-        expect(response.body).to eq(project.stories.to_json)
+      context 'when responding to json' do
+        specify do
+          get :index, xhr: true, params: { project_id: project.id }
+
+          expect(response).to be_successful
+          expect(response.body).to eq(project.stories.to_json)
```

Comment:
prefer to keep on expect per test to get better feedback and easy debug
http://www.betterspecs.org/#single